ngmix/priors/random.py

```diff
@@ -0,0 +1,27 @@
+import numpy
+
+
+def make_rng(rng=None):
+    """
+    if the input rng is None, create a new RandomState
+    but with a seed generated from current numpy state
+    """
+    if rng is None:
+        seed = numpy.random.randint(0, 2 ** 30)
+        rng = numpy.random.RandomState(seed)
+
+    return rng
+
+
+def srandu(num=None, rng=None):
```

Comment:
should remove and require the functions that use it to use the sent rng, and require that rng is not None